torch/hub.py

```diff
@@ -158,13 +143,15 @@ def _get_cache_or_reload(github, force_reload):
     use_cache = (not force_reload) and os.path.exists(repo_dir)
 
     if use_cache:
-        sys.stderr.write('Using cache found in {}\n'.format(repo_dir))
+        if verbose:
+            sys.stderr.write('Using cache found in {}\n'.format(repo_dir))
     else:
         cached_file = os.path.join(hub_dir, branch + '.zip')
         _remove_if_exists(cached_file)
 
         url = _git_archive_link(repo_owner, repo_name, branch)
-        _download_archive_zip(url, cached_file)
+        sys.stderr.write('Downloading: \"{}\" to {}\n'.format(url, cached_file))
```

Comment:
Do we want to guard this under the verbose argument?

Reply:
Yea I thought about this as well, and I think it might be better to keep it so that users know that we are downloading stuff as a one-time cost. This is also the preferred way from user comment here https://github.com/pytorch/pytorch/issues/24791#issuecomment-522764313. 
We can definitely change it if needed. :D